config.json

```
@@ -13,7 +13,7 @@
         "sync": { "interval": 1 },
         "school": {
             "location": "/usr/app/custom-modules/school-module",
-            "schoolName": "My School",
+            "schoolName": "ISS - International Schul Schule",
```

As it is written in the `school_logo.png`.

```suggestion
            "schoolName": "ISS - Internationale Schul Schule",
```